agent/lib/kontena/workers/fluentd_worker.rb

```diff
@@ -44,13 +44,10 @@ def on_node_info(topic, info)
     end
 
     def on_log_event(topic, log)
-      # TODO Get more tags
-      # maybe stack.service.instance
-      #puts "forwarding: #{@forwarding}, fluentd: #{@fluentd}"
       if @forwarding && @fluentd
         tag = [log[:stack], log[:service], log[:instance]].join('.')
         record = {
-          event: log[:data], # the actual log event
+          log: log[:data], # the actual log event
```

Comment:
Also needs update to the docs `{ event: <log data> }` bit

:+1: for conforming to the Docker fluent logging driver